xbmc/platform/darwin/ios/XBMCController.mm

```diff
@@ -30,7 +32,13 @@
 #include "utils/URIUtils.h"
 #include "utils/log.h"
 #include "windowing/XBMC_events.h"
-#include "windowing/ios/WinSystemIOS.h"
+#import "windowing/ios/WinSystemIOS.h"
+
+#import "platform/darwin/NSLogDebugHelpers.h"
+#import "platform/darwin/ios/IOSEAGLView.h"
+#import "platform/darwin/ios/IOSScreenManager.h"
+#import "platform/darwin/ios/XBMCApplication.h"
+#import "platform/darwin/ios-common/DarwinEmbedNowPlayingInfoManager.h"
```

Comment:
This is at the wrong place, `platform/darwin/ios-` has to be moved before `platform/darwin/ios/`

Reply:
It is common for tvos. Can it stay here, to make the tvos PR easier

Reply:
If you want a different format then clang-format uses, open a PR which adjusts the configuration. Otherwise follow the formatting.

Reply:
Sorry, misread the statement, i thought the complaint was about the file placed in ios-common instead of ios, no problems. Will update